test/index.spec.js

```diff
@@ -3147,6 +3147,15 @@ describe('unexpected-dom', () => {
       );
     });
 
+    it('supports searching for a plain string', () => {
+      expect(
+        '<div><span class="greeting"><span>Hello</span><span class="name">Jane Doe</span></span></div>',
+        'when parsed as HTML',
+        'to contain',
+        'Jane Doe'
+      );
+    });
```

Comment:
Hmm, on second thought this might be ambiguous -- is this supposed to pass?

```js
expect('<div><foo>bar</foo></div>', 'to contain', '<foo>bar</foo>')
```

or this?

```js
expect('<div>&lt;foo&gt;bar&lt;/foo&gt;</div>', 'to contain', '<foo>bar</foo>')
```

With this change I think only the latter will pass? As a use I think I'd expect it not to. I think the RHS string should be interpreted as HTML, not the value of a text node.

Reply:
Oooh good catch - is this a hint we need to take the node type into account somehow?

Reply:
I just tried it, the later fails. I agree - let's remove it again.